config/rbac/tektonconfig_operator_rolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tektonconfig-operator-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tektonconfig-operator-role
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: system
```

Comment:
Binding to the `default` system service account? Don't we want to bind to the operator's SA?

Reply:
there was no operator SA previously created in this repo nor in the tekton operator repo

See the tekton deployment (no SA specified):  https://github.com/tektoncd/operator/blob/937dea56d9ed6011b1b3c8de4ed43c1cc4cba38b/config/kubernetes/operator.yaml
and   
https://github.com/tektoncd/operator/blob/937dea56d9ed6011b1b3c8de4ed43c1cc4cba38b/config/openshift/operator.yaml

https://github.com/shipwright-io/operator/blob/main/config/manager/manager.yaml

I took adding an SA as out of scope for this particular Jira

Reply:
ok @adambkaplan and I talked in office hours ... when  I looked at the file, saw I merged tekton operator vs. ours

now, we are currently following the tekton operator's conversion of using the `default` SA in the `system` namespace

@adambkaplan and I agreed we won't change things with this PR.  But he will be opening upstream issues / Jira's for the future work cited in this PR, and created a non-default SA for our operator will be one of those items.